Makefile

```diff
@@ -0,0 +1,35 @@
+all: cloudtrail_streamer
+
+# Package lambda function in zip file
+package:
+	docker run -i --rm -v `pwd`:/go/src/github.com/mozilla-services/cloudtrail-streamer \
+		golang:1.10 \
+		/bin/bash -c 'cd /go/src/github.com/mozilla-services/cloudtrail-streamer && make lambda'
```

Comment:
does `golang:1.10` give you `1.10` or the latest in the `1.10.x` branch? If it's `1.10`, I'd suggest using `1.10.3` instead.

Reply:
Latest in the `1.10.x` branch.

```
$ docker run golang:1.10 go version
go version go1.10.3 linux/amd64
```